hlpr_feature_extraction/include/utils_pcl_ros.hpp

```diff
@@ -43,14 +47,32 @@ Box3D
 //boundingBoxWithCoeff(pcl::PointCloud<PointT> &cluster, pcl::ModelCoefficients::Ptr coefficients);
 boundingBoxWithCoeff(pcl::PointCloud<PointT> &cluster, pcl::ModelCoefficients::Ptr coefficients, pcl::PointCloud<PointT>::Ptr &cloud_transformed);
 
+//void
+//fillRosMessage (hlpr_feature_extraction::PcFeatures &outRosMsg, const pc_cluster_features &inObjFeatures);
```

Comment:
Again, can probably delete this and the above boundingBoxWithCoeff comment

Reply:
Delete this as it is a reference to the old message types and organization.